src/Widget/TwigAware.php

```diff
@@ -16,4 +16,8 @@ interface TwigAware
     public function setTwig(Environment $twig): WidgetInterface;
 
     public function getTwig(): Environment;
+
+    public function getTemplate(): string;
```

Comment:
No, these are not part of twig aware either. They exist only in BaseWidget.

Reply:
Removed